lib/dogma/rules/final_newline.ex

```diff
@@ -8,7 +8,9 @@ defmodule Dogma.Rules.FinalNewline do
   alias Dogma.Script
   alias Dogma.Error
 
-  def test(script) do
+  def test(script), do: test(script, [])
+
+  def test(script, _config = []) do
```

Comment:
@lpil yeah I wasn't keen on the fact I've added all this boilerplate code for rules. Currently I haven't thought of anything neater than having another function like valid_config?(config).